Examples/MAX78000/EEPROM_Emulator/project.mk

```diff
@@ -0,0 +1,11 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://github.com/Analog-Devices-MSDK/VSCode-Maxim/tree/develop#build-configuration
+
+# **********************************************************
+
+# Add your config here!
+BOARD=FTHR_RevA
```

Comment:
Should be compatible with the EVKIT as well right?

Reply:
Yes. I forgot to take this out. Gone now though, should default to the EVKIT.